java/code/src/com/redhat/rhn/manager/audit/ScapManager.java

```diff
@@ -70,6 +67,8 @@
 import java.util.Map;
 import java.util.Optional;
 
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
```

Comment:
Hmm, didn't I just read that this was deprecated again in Java 17 ?
Or is this an external jar?

Reply:
Yes, standard is the wrong word here, it was more in the sense of base, plain :slightly_smiling_face: . 

But indeed, JAXB was part of Java EE which no longer exists, and now is part of jakarta so when we update the jar we will need to update the package names. I wanted to migrate away from `simple-xml` since it's a 10 years old unmaintened library. We can also investigate other frameworks, but jaxb is currently part of our dependencies so I didn't have to add add anything new:

```
        <dependency org="suse" name="jaxb-runtime" rev="2.3.1" />
        <dependency org="suse" name="jaxb-impl" rev="2.3.1" />
```

Reply:
Let's hope the best that this provides everything what we need. This is the only path I found in the jar which I know :-)
glassfish-jaxb/jaxb-impl.jar

      35  Defl:N       37  -6% 2024-01-12 06:58 2499855a  META-INF/services/javax.xml.bind.JAXBContext